getting-started.adoc

```diff
@@ -1,3 +1,229 @@
+
+[NOTE]
+.What if there are no outstanding offers for the payment method I want to use?
+====
+If you don't see any offers for the payment method you want to use, consider checking back later. A Bisq offer is only available while the user making the offer is online and running Bisq, so you'll see offers come and go as Bisq users around the world go online and offline throughout the day.
+
+You can also create your own buy offer with the "Create New Offer to Buy BTC with USD" button on the bottom-right, but we won't go into those details here.
+====
+
+[NOTE]
+.Where is my payment account data stored?
+====
+On your computer *only* (i.e., the computer on which Bisq is installed). Only your trading partner (and your arbitrator, if there's a dispute) will ever see your payment details.
+====
+
+In this tutorial, we're going to configure a Zelle account. You can adapt the instructions that follow to the payment method you choose.
+
+Go to the Account panel and click "Add New Account." Pick Zelle from the dropdown and enter your details. 
+
+For Zelle, we need to enter Full Name and Email/Mobile Number.
+
+.Configuring a Zelle account.
+image::configure-fiat-account.png[Configuring a national currency account]
+
+You don't need to edit the "Salt" field. You can keep the "Account Name" unchanged or make it whatever you like: it's just a label only you will ever see, so there's no right or wrong value for it.
+
+IMPORTANT: It's *crucial* the information you enter here is correct. Your trading partner can only acknowledge your payment if it's from the account detailed here. Once you've created an account, you cannot edit its information (if you need to make a change, you can delete & add the account again).
+
+Once you're done, hit the "Save New Account" button and go back to the Buy BTC panel. Offers requiring the payment method you just configured should now be enabled:
+
+.After configuring a payment account: we picked Zelle, so Zelle offers are now enabled.
+image::after-adding-fiat-account.png[Configuring a national currency account]
+
+== Get a security deposit
+
+Just 1 more thing to do before we can trade with Bisq: we need to obtain enough bitcoin for a security deposit.
+
+To prevent fraud, Bisq requires buyers and sellers to post a security deposit in bitcoin for every trade. On the buy side, it's {btc-deposit} BTC, along with a trading fee & mining fees for the transfers.
+
+As of this writing, that means you'll need {btc-deposit-plus-fees} BTC.
+
+If you don't already own this bitcoin, you'll need to obtain it. How? You've got options. Do you have any friends or family who could help? Otherwise, you can try a local bitcoin ATM (many don't require ID for small transactions), vouchers, or bitcoin meetups/events.
+
+[WARNING]
+.How about getting bitcoin through an exchange like Coinbase?
```

Comment:
Coinbase is not an exchange. It's a digital wallet and platform. GDAX is an exchange operated by Coinbase.